.github/ISSUE_TEMPLATE/icon_update.yml

```diff
@@ -0,0 +1,67 @@
+name: Icon update
+description: Help us improve by reporting outdated icons
+title: "Update: "
+labels: [icon outdated]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `icon outdated`
+        label][icon-outdated-issues] (including closed issues) before you continue. If
+        you find one for the brand you're reporting then leave a comment on it or add
+        a reaction. For more details see the [Contributing
+        Guidelines][contributing-guidelines].
+
+
+        [contributing-guidelines]:
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+
+        [icon-outdated-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22icon+outdated%22+is%3Aopen
```

Comment:
The current formatting looks a bit odd to me, I believe this suggestion still complies with the formatting you're aiming for.

Also, the same `[Contributing Guidelines]` and `[x]: y` suggestions as before.

```suggestion
        Before opening a new issue, make sure it isn't covered by an existing issue.
        Please search for [issues with the `icon outdated` label][icon-outdated-issues]
        (including closed issues) before you continue. If you find one for the brand
        you're reporting then leave a comment on it or add a reaction. For more details
        see the [Contributing Guidelines].


        [contributing guidelines]: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md

        [icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22icon+outdated%22+is%3Aopen
```

Additionally, the link to the contributing guidelines feels a bit out of place here, if you agree:

```suggestion
        Before opening a new issue, make sure it isn't covered by an existing issue.
        Please search for [issues with the `icon outdated` label][icon-outdated-issues]
        (including closed issues) before you continue. If you find one for the brand
        you're reporting then leave a comment on it or add a reaction.


        [icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22icon+outdated%22+is%3Aopen
```

Reply:
I agree